src/Compilers/CSharp/Portable/Binder/SimpleLocalScopeBinder.cs

```diff
@@ -21,5 +22,15 @@ protected override ImmutableArray<LocalSymbol> BuildLocals()
         {
             return _locals;
         }
+
+        internal override ImmutableArray<LocalSymbol> GetDeclaredLocalsForScope(CSharpSyntaxNode scopeDesignator)
```

Comment:
This binder is not used by the compiler and is never associated with any scope in source.